### PR TITLE
Private source

### PR DESCRIPTION
**Pull request by karitotp**
cc. @Rub21

### CODE REVIEW

File: validators/privateSource/map.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+var turf = require('turf');
+
+module.exports = function(tileLayers, tile, writeData, done) {
+  var layer = tileLayers.osm.osm;
+  var osmlint = 'privatesource';
+  var result = [];
+  var unallowedSource = {
+    'Google Maps': true,
```
Comment (maning) [inline validators/privateSource/map.js]: Can we make this more generic like any strings with the Google?

File: validators/privateSource/map.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'Google Satellite': true,
+    'Google Earth': true,
+    'Google Street View': true,
+    'Waze': true,
+    'ERE Maps': true,
```
Comment (maning) [inline validators/privateSource/map.js]: should be `Here`

**Comment by geohacker** [comment]
@Rub21 @karitotp - thank you for writing this! It'll be interesting to run this on planet. 
I agree with @maning - can we make the strings more generic? May be use regular expressions? We do similar check on changeset metadata level on OSMCha https://github.com/mapbox/osmcha/blob/master/osmcha/changeset.py#L293-L302

**Comment by Rub21** [comment]
we are ready here https://gist.github.com/anonymous/c2f96cea5932ee55cb1b665bf8ea9e6c, @maning  can you check the output for argentina,  

@geohacker @maning , what do you think?, if we run this validator for whole the world and created at a task in to-fix.

**Comment by maning** [comment]
Detection is great.  My main concern is how this should be fixed.  For example ,the changeset for a node in the sample data has other issues. See source tag: http://www.openstreetmap.org/changeset/19622042

I'm not sure this should be a to-fix task.  Such edits should handled at the changeset level.  Specific actions should be:
- reach the mapper via changeset discussion;
- if confirmed to be of questionable source, revert changeset (partial/full).

**Comment by geohacker** [comment]
@maning I agree with your observations. Not seeing value in pushing this out as a task on to-fix.

**Pull request merged**